drools-parser/src/test/java/org/drools/parser/MiscDRLParserTest.java

```diff
@@ -57,36 +57,47 @@ private String readResource(final String filename) throws Exception {
         return sb.toString();
     }
 
+    private RuleDescr parseAndGetFirstRule(String drl) {
+        PackageDescr pkg = parser.parse(drl);
+        assertThat(parser.hasErrors()).as(parser.getErrorMessages().toString()).isFalse();
+        assertThat(pkg.getRules()).isNotEmpty();
+        return pkg.getRules().get(0);
+    }
+
+    private RuleDescr parseAndGetFirstRuleFromFile(String filename) throws Exception {
+        return parseAndGetFirstRule(readResource(filename));
+    }
+
     @Test
-    void testPackage() {
+    void validPackage() {
         final String source = "package foo.bar.baz";
         final PackageDescr pkg = parser.parse(source);
         assertThat(pkg.getName()).isEqualTo("foo.bar.baz");
     }
 
     @Test
-    void testPackageWithErrorNode() {
+    void packageWithErrorNode() {
```

Comment:
As in previous comments, IMO the name of the test should indicate a) the method actually testend AND b) the condition that is tested.

Reply:
@gitgabrio Agreed. I'll follow the practice. Thanks!